lib/full-site-editing/edit-site-page.php

```diff
@@ -152,7 +152,7 @@ static function( $classes ) {
 		'siteUrl'                              => site_url(),
 		'postsPerPage'                         => get_option( 'posts_per_page' ),
 		'styles'                               => gutenberg_get_editor_styles(),
-		'defaultTemplateTypes'                 => gutenberg_get_indexed_default_template_types(),
+		'defaultTemplateTypes'                 => get_default_block_template_types(),
```

Comment:
Gutenberg plugin is using a special function here that is different from Core, I was not entirely sure which one was the right one, but I decided to use the Core approach. Let me know if this is wrong.

Reply:
There's a failing end2end test, if I had to guess, I think it's related to the change here, meaning Core is probably also broken and missing a backport :)

Reply:
Ok so this should be solved with dd377ed and that commit need to be backported in Core properly.